src/js/components/rating.js

```diff
@@ -0,0 +1,30 @@
+import Raty from 'raty-js'
+
+export default els => {
+  for (const el of els) {
+    let options
+
+    try {
+      options = JSON.parse(el.dataset.ofRating || '{}')
+    } catch (error) {
+      console.error(error) // eslint-disable-line no-console
+      options = {}
+    }
+
+    const rating = new Raty(el, {
+      ...options,
+      starType: 'i',
+      click(number) {
+        const event = new CustomEvent('clicked.of.rating', {
```

Comment:
@qzminski I think all `OpenFrontend` reladed events should start with `of.` so it acts like a namespace. So this should be `of.rating.clicked` in my opinion. Going from the widest scope (OF) to the component scope (Rating) to the action (clicked) 😊

Reply:
We thought so, too, but if we are to stick to Bootstrap convention then we have to leave it like that.

https://getbootstrap.com/docs/5.3/components/modal/#events

Reply:
Super weird design decision 😅 but I agree that this would make sense then 👍